.github/workflows/dco.yml

```diff
@@ -7,15 +7,4 @@ on:
 
 jobs:
   commits_check_job:
-    runs-on: ubuntu-latest
-    name: Commits Check
-    steps:
-    - name: Get PR Commits
-      id: 'get-pr-commits'
-      uses: tim-actions/get-pr-commits@master
-      with:
-        token: ${{ secrets.GITHUB_TOKEN }}
-    - name: DCO Check
-      uses: tim-actions/dco@master
-      with:
-        commits: ${{ steps.get-pr-commits.outputs.commits }}
+    uses: nspcc-dev/.github/.github/workflows/dco.yml@master
```

Comment:
But still not sure any CI that depends on relative label (`master`) is a good practice.

Reply:
This one certainly is. There are warnings on https://github.com/nspcc-dev/neofs-dev-env/actions/runs/6013977580?pr=284 but they'll be fixed in one place for all repositories.